examples/applications/author-disambiguation/learning_model.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
```

Comment:
What about `distance.py` for the filename? `model` is a bit vague, dont you think?